src/Schema.php

```diff
@@ -800,13 +800,9 @@ protected function loadColumnSchema(array $info): ColumnSchemaInterface
 
         if (
             $defaultValue !== null
-            && preg_match("/nextval\\('\"?\\w+\"?\.?\"?\\w+\"?'(::regclass)?\\)/", $defaultValue) === 1
+            && preg_match("/^nextval\('([^']+)'(?:::regclass)?\)$/", $defaultValue, $matches) === 1
```

Comment:
```suggestion
            && preg_match("/^nextval\('([^']+)'(::regclass)?\)$/", $defaultValue, $matches) === 1
```

Can you add test with `::regclass` ?